Source/WebKit/UIProcess/Automation/mac/WebAutomationSessionMac.mm

```diff
@@ -101,9 +101,6 @@
     for (NSEvent *event in eventsToSend) {
         LOG(Automation, "Sending event[%p] to window[%p]: %@", event, window, event);
 
-        // Take focus back in case the Inspector became focused while the prior command or
-        // NSEvent was delivered to the window.
-        [window makeKeyAndOrderFront:nil];
```

Comment:
Is removing this necessary. It still seems like the original comment would apply if the application is active but the window we are targeting isn't front and center yet?

Reply:
Correct, removing this is not necessary. Also, given that my fix doesn't address the bug with modifier-included keyboard events, we could put this back so we at least make an attempt to focus the window so that those events work.